### ISSUE TITLE
Revenant dust in someones butt teleport glitch 

### ISSUE THREAD

**Issue by lewismaaate** [HippieStation/HippieStationdeprecated2020#8921]
When a revenant is killed and someone takes the dust and then puts it into their butt, when the revenant respawns it is taken to the super secret area with the weird crystals.

**Comment by JohnGinnane** [comment]
Sounds like an issue with inventory component @Carbonhell

**Comment by ghost** [comment]
what super secret area?  what are the coders hiding?

**Comment by ghost** [comment]
sending them to 0,0,0 most likely
the question is if it applies to other storage stuff like storage implants

**Comment by JohnGinnane** [comment]
@zamolxius a while back /tg/ turned storage into a component so any items in the game could be used for storage, but it means when you place an item into "storage" the location is set to something like "0, 0, 0" as McBaw says

I'm not actually sure why they do it that way but I think lockers were the same (might still be the same?)

**Comment by lewismaaate** [comment]
next time you observe or are ghost, tele to the location im pretty sure its titled super secret area

**Comment by ghost** [comment]
yeah, like when standing in a locker made you go into 0.0.0's temp and freeze

**Comment by Eagle-Eyess** [comment]
I want to know who kills a revenant and then decides to shove their basically dead corpse up their ass...
